### ISSUE TITLE
Can DbSchema Support Foreign Table or 'Multiple Database Connections' 

### ISSUE THREAD

**Issue by ylyxf** [wise-coders/dbschema#58]
In Oracle Database , there are some foreign tables created by synonyms based on database links,  I want show these table in DbSchema Layouts.

But the connection can only reverse Tables , Views, User Data Types, Squeences, Procedures,Functions,Triggers.
It can not  reverse Synonyms , so I can't use 'Foreign Tables' in Laoyout。

Is it possiable to change the conception of 'Connection' in DbSchema form **only one** Database Connetion to **a group of** Database Connections, then one Schema/Layout can contains different tables from different Databases.

**Comment by wise-coders** [comment]
This is an interesting idea. We have in plan to add multi-model projects, each model with its connections. Then this will be possible. Is a matter of time to get it done.

Best regards,
 Dragos Pruteanu

**Comment by brettshearer** [comment]
I would be interested in being able to split a model simply to reduce the size of the generated dbs file. Mine is currently 16MB and I'm looking to move to DBschema due to the performance of the previous CASE tool I used.
Would prefer to split the model up (yet still eventually deploy it monolithically for now). This introduces virtual foreign keys which would actually be real foreign keys in the production database.

**Comment by wise-coders** [comment]
Please let us know which operations are slow right now. Maybe we can focus and improve this first, and let DbSchema as it is.
A short video would help. Also reducing the number of layouts could help.

Which database are you using? If the database supports virtual foreign keys, we can also add this option to the FK options, so you can create the virtual foreign keys in the database. Please send me a sample SQL command to create the virtual fk in the database.

Best regards,
 Dragos Pruteanu

**Comment by brettshearer** [comment]
I don’t believe I’ve reported any slowness – everything is actually really fast and I impressed with the overall speed of all functions.

**Comment by wise-coders** [comment]
I got it. I read this message:
I would be interested in being able to split a model simply to reduce the size of the generated dbs file. Mine is currently 16MB and I'm looking to move to DBschema due to the performance of the previous CASE tool I used.

and I thought the performance issues are related to slowness. Which is then the reason to reduce the size of the dbs file?
Even if the file is relative large, this shouldn't be a problem.

Best regards,
 Dragos Pruteanu
